### ISSUE TITLE
Error when compiling

### ISSUE THREAD

**Issue by CeciliaPanicali** [shuding/nextra#61]
I started receiving this error on a Nextra project: I tried deleting almost any file and the error is still there (either with Node 12.x or 14.x).
I tried also removing the `.next` folder, the `node_modules` folder and the `yarn.lock` file and reinstalling modules, but I still get the error.

The repo is here:
https://github.com/ReactBricks/react-bricks-docs

Thank you,
Cecilia 

```
(node:11513) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '1' of null
    at removeExtension (/Users/cecilia/Documents/Development/react-bricks-docs-v2-nextra/node_modules/nextra/dist/loader.js:70:32)
    at /Users/cecilia/Documents/Development/react-bricks-docs-v2-nextra/node_modules/nextra/dist/loader.js:90:42
    at Array.map (<anonymous>)
    at getFiles (/Users/cecilia/Documents/Development/react-bricks-docs-v2-nextra/node_modules/nextra/dist/loader.js:88:44)
    at async getPageMap (/Users/cecilia/Documents/Development/react-bricks-docs-v2-nextra/node_modules/nextra/dist/loader.js:143:11)
    at async Object.loader (/Users/cecilia/Documents/Development/react-bricks-docs-v2-nextra/node_modules/nextra/dist/loader.js:177:26)
(node:11513) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:11513) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by shuding** [comment]
Strange, I successfully built the repository using `yarn build`:

![CleanShot 2020-12-17 at 20 01 12@2x](https://user-images.githubusercontent.com/3676859/102485721-f426df80-40a2-11eb-823d-63db611c7f1f.png)

But from the error message you shared, I guess that there's a file inside `/pages` which doesn't have an extension in its file name. 

I'll fix that function to make it work in that case! Thanks for reporting.

**Comment by CeciliaPanicali** [comment]
That's really strange: I am sure no file was without extension and I finally made it work by deleting the whole project and cloning it again. 
Thank you :)

**Comment by gsimone** [comment]
This happened to me when OSX created a .DS_Store in the `pages` dir, solved by removing it

**Comment by shuding** [comment]
Thanks @gsimone! That makes totally sense, will fix it.

**Comment by shuding** [comment]
Fixed in #62, thanks to @fanweixiao. Also released `nextra@0.3.2`.
